diff --git a/chrome/browser/ui/browser.cc b/chrome/browser/ui/browser.cc
index 17f2591ac638..4dd930dde502 100644
--- a/chrome/browser/ui/browser.cc
+++ b/chrome/browser/ui/browser.cc
@@ -226,6 +226,7 @@
 #include "url/origin.h"
 #include "url/scheme_host_port.h"

+#include "brave/browser/ui/tabs/brave_tab_strip_model.h"
 #if defined(OS_WIN)
 #include <shellapi.h>
 #include <windows.h>
@@ -413,7 +414,7 @@ Browser::Browser(const CreateParams& params)
       tab_strip_model_delegate_(
           std::make_unique<chrome::BrowserTabStripModelDelegate>(this)),
       tab_strip_model_(
-          std::make_unique<TabStripModel>(tab_strip_model_delegate_.get(),
+          std::make_unique<BraveTabStripModel>(tab_strip_model_delegate_.get(),
                                           params.profile)),
       app_name_(params.app_name),
       is_trusted_source_(params.trusted_source),
