diff --git a/chrome/browser/ui/browser.cc b/chrome/browser/ui/browser.cc
index d262a424f49b2299a3367a265299a4c74328e4d6..964724e92442c315c53228784016eba9d94da041 100644
--- a/chrome/browser/ui/browser.cc
+++ b/chrome/browser/ui/browser.cc
@@ -233,6 +233,7 @@
 #include "url/origin.h"
 #include "url/scheme_host_port.h"

+#include "brave/browser/ui/tabs/brave_tab_strip_model.h"
 #if defined(OS_WIN)
 #include <shellapi.h>
 #include <windows.h>
@@ -412,7 +413,7 @@ Browser::Browser(const CreateParams& params)
       tab_strip_model_delegate_(
           std::make_unique<chrome::BrowserTabStripModelDelegate>(this)),
       tab_strip_model_(
-          std::make_unique<TabStripModel>(tab_strip_model_delegate_.get(),
+          std::make_unique<BraveTabStripModel>(tab_strip_model_delegate_.get(),
                                           params.profile)),
       app_name_(params.app_name),
       is_trusted_source_(params.trusted_source),
